cmd/kubeadm/app/cmd/phases/workflow/phase.go

```diff
@@ -24,6 +24,9 @@ type Phase struct {
 	// the same workflow or phases belonging to the same parent phase).
 	Name string
 
+	// Aliases returns the aliases for the phase.
+	Aliases []string
+
```

Comment:
are Aliases part of the scope of this PR?
best to have them separately.

Reply:
Yes, this phase has a aliase. wdyt?

Reply:
i will leave this to @fabriziopandini for a comment.

Reply:
I'm ok with adding aliases in this because it is a small changes